Lib/fontmake/__main__.py

```diff
@@ -31,6 +31,9 @@ def main():
     parser.add_argument('--family-name', help='Family name to use for masters,'
                         'and to filter output instances by')
     parser.add_argument('--use-afdko', action='store_true')
+    parser.add_argument('--keep-direction', dest="reverse_direction",
+                        action='store_false', help='Do not reverse contour '
+                        'direction')
```

Comment:
or maybe make sure cu2qu does not modify the input fonts unless it finds a cubic curve segment.
The GetSegmentsPen that gathers the contour data could be modified to also record whether a glyph contains any curve or qcurve. This way one could skip the cu2qu conversion if it's not necessary.
https://github.com/googlei18n/cu2qu/blob/master/Lib/cu2qu/ufo.py#L97